system_test/aest_heavy_sync_SUITE.erl

```diff
@@ -0,0 +1,184 @@
+-module(aest_heavy_sync_SUITE).
```

Comment:
I ran this test against master version of the node and I got failure (that is good):
```
Node node2's log "epoch.log" contains errors:
2018-10-24 09:29:23.623 [error] <0.1506.0> CRASH REPORT Process <0.1506.0> with 0 neighbours exited with reason: {timeout,{gen_server,call,[aec_conductor,{add_synced_generation,#{add_keyblock => true,dir => forward,key_block => {key_block,{key_header,3...
```